addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -447,17 +459,40 @@ Object.assign(FeaturesSelectionScreen, {
     template: 'website.Configurator.FeatureSelection',
 });
 
+
 class ThemeSelectionScreen extends ApplyConfiguratorScreen {
     setup() {
+        const uiService = useService('ui');
         super.setup();
 
         this.orm = useService('orm');
         this.state = useStore();
         this.themeSVGPreviews = [useRef('ThemePreview1'), useRef('ThemePreview2'), useRef('ThemePreview3')];
+        this.promiseTab = [];
```

Comment:
defs, proms, promises, please use same name as everywhere else in the code :)

Also, I'd not go with `this` here, it's not used elsewhere, except here right?